### DISCUSSION TITLE
Create a collection from opened tabs

### DISCUSSION THREAD

**Discussion by mjthoraval** [windingwind/zotero-actions-tags#213] [Q&A]
I would like to create automatically a new collection containing all the opened tabs.
The name of the collections should be the date, e.g.: `2023_11_30`.
It should be a sub-collection of a general collection used to store tabs, so that the number of first level collections does not clutter the collections pane. For example: `@Tabs`.
If the collection already exists, it should just move the items corresponding to the opened tabs to the existing collection.

This would be complementary to this other script that allows moving the opened tabs to a note: https://github.com/windingwind/zotero-actions-tags/discussions/206
That script is very nice if I want to take notes on the items I have opened.
But for long term, it may be easier to organize the items in a collection, so that I can continue adding or removing items manually from that collection.

**Comment by mjthoraval** [comment]
Related Forums discussions:
https://forums.zotero.org/discussion/105993/is-there-a-way-to-save-a-session-save-the-open-files-in-the-zotero-pdf-viewer-in-a-subcollection
https://forums.zotero.org/discussion/112253/saving-open-tabs-and-show-file

**Comment by Abraxas-Knister** [comment]
I find relatively often that I miss this feature and decided now was the time to leave a note about it.

**Comment by Geo123abc** [comment]
This may be what you need  [[Share] Working script: save all open Reader tabs as a timestamped session · windingwind/zotero-actions-tags · Discussion #531](https://github.com/windingwind/zotero-actions-tags/discussions/531)
